packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -16,10 +16,11 @@ import '../info_controller.dart';
 class InfoScreen extends Screen {
   const InfoScreen()
       : super(
-          DevToolsScreenType.info,
+          id,
           title: 'Info',
           icon: Octicons.info,
         );
+  static const id = 'info';
```

Comment:
as in the other cases, remove this const and write the value inline in the constructor.